pages/api/team-member/[mail].tsx

```diff
@@ -0,0 +1,12 @@
+import prisma from '../../../lib/prisma'
+
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+    const teamMemberMail = req.query.mail.toString()
```

Comment:
we need permissions check here. If user is not admin, this api should return an error. I think there is a similar logic for user generation. Also, afterwards, it needs a unit test.